### ISSUE TITLE
Improved and customizable front page

### ISSUE THREAD

**Issue by DogCatPuppyLover** [ScratchAddons/ScratchAddons#6082]
### Why this would be helpful

The "What's Happening?" section on the Front Page is the only place where you can see what Scratchers you follow are doing. It's rather small, unlike many other websites, where the activity of those you follow is displayed prominently. So, I propose an addon that renovates the front page to make it more like a dashboard of the activity of Scratchers you follow. This would make it easier to discover new content, and connect with your friends. It would also be helpful if you could "track" certain studios, like studios that attempt to curate a select number of good projects, and see any project that gets added to them on the front page!

### How this addon works

This addon would allow people to customize the front page, by removing rows that they don't need and adding new ones. Here are some ideas for new rows:

- New Projects from Scratchers I Follow: This would show the new projects from Scratchers that you follow. This would work by loading a bunch of "What's happening" activity and filtering it.
- New Projects from Studios I'm Subscribed To: This would add a "subscribe" feature to studios, and at regular intervals (like once per day, or once per hour--this can be customized by the user) it would check all of these studios for new projects, and display the new projects here. This way, you could 
- Who Scratchers I Follow are Following: This would show the people that the Scratchers you follow have recently followed. This would work similar to "New Projects from Scratchers I Follow".

Here's a quick mockup:

![Screenshot 2023-05-20 at 08-59-11 Scratch - Imagine Program Share](https://github.com/ScratchAddons/ScratchAddons/assets/79656307/9d571e0b-9d9c-497c-a47b-ab7a25505e17)

### Possible alternatives

Things like "New Projects from Scratchers I Follow" seem like a good thing to have in vanilla Scratch, but considering how infrequently the ST actually takes suggestions, I don't think that suggesting it in the forums is going to be helpful, at least in the short term.

**Comment by DogCatPuppyLover** [comment]
This may be a duplicate of #1758, or at least very similar.

**Comment by Secret-chest** [comment]
> This would add a \"subscribe\" feature to studios

Following studios exists, and does nothing.

**Comment by BroJac5246** [comment]
> This may be a duplicate of #1758, or at least very similar.

I think this issue should be part of that, as they would probably wind up in the same addon.

**Comment by WorldLanguages** [comment]
> New Projects from Scratchers I Follow

Didn't this row exist for years...? Mandela effect?

**Comment by WorldLanguages** [comment]
> This may be a duplicate of #1758, or at least very similar.

I think this issue can stay separate as it explicitly suggests using the data from "What's Happening?".

**Comment by DogCatPuppyLover** [comment]
> > This would add a "subscribe" feature to studios
> 
> Following studios exists, and does nothing.

Yes, but the problem with that is that many people already follow hundreds of studios, and might not want to "subscribe" to all of them, or unfollow the ones they don't want to subscribe to. It would be best if it starts as a blank slate. The studios that you subscribe to can be stored in the extension data storage.

**Comment by medlans** [comment]
> > New Projects from Scratchers I Follow
> 
> Didn't this row exist for years...? Mandela effect?

[Yeah it did, my brain told me it still existed though](https://en.scratch-wiki.info/wiki/Projects_by_Scratchers_I%27m_Following).

Is What the Community is Sharing possible?

**Comment by WorldLanguages** [comment]
> [Yeah it did, my brain told me it still existed though](https://en.scratch-wiki.info/wiki/Projects_by_Scratchers_I%27m_Following).

Quote from that Wiki page:

> Older Scratchers can still see this row, however it is no longer updated.

But my row has some recent projects! 🤔

**Comment by WorldLanguages** [comment]
> Is What the Community is Sharing possible?

Probably not, unless we show the projects from https://scratch.mit.edu/explore/projects/all/recent but in a row.

**Comment by BroJac5246** [comment]
> > > This would add a "subscribe" feature to studios
> > 
> > Following studios exists, and does nothing.
> 
> Yes, but the problem with that is that many people already follow hundreds of studios, and might not want to "subscribe" to all of them, or unfollow the ones they don't want to subscribe to. It would be best if it starts as a blank slate. The studios that you subscribe to can be stored in the extension data storage.

I disagree; the whole point of following studios is to... (dun dun dun!) follow studios (more specifically, the studios' activity)! If a user chooses to use the feature in a way that they're not intended to then that's up to them, but they should expect features centered around the intended use: following activity in a studio. We should not introduce a feature that is literally a clone of a pre-existing feature.

**Comment by mxmou** [comment]
> > [Yeah it did, my brain told me it still existed though](https://en.scratch-wiki.info/wiki/Projects_by_Scratchers_I%27m_Following).
> 
> Quote from that Wiki page:
> 
> > Older Scratchers can still see this row, however it is no longer updated.
> 
> But my row has some recent projects! 🤔

This post explains what's wrong with those rows: https://scratch.mit.edu/discuss/post/3524962/

**Comment by Secret-chest** [comment]
Look, Projects in Studios I'm Following also existed!
https://en.scratch-wiki.info/wiki/Projects_in_Studios_I%27m_Following

**Comment by Hans5958** [comment]
#

> > This may be a duplicate of #1758, or at least very similar.
> 
> I think this issue can stay separate as it explicitly suggests using the data from "What's Happening?".

I don't think it warrants for this issue to be separate. I suggest merging it by closing it and add the suggestions there.
